### ISSUE TITLE
obs-asio doesn't have a version number after windows install

### ISSUE THREAD

**Issue by leochras** [Andersama/obs-asio#103]
**Describe the bug**
There is a manifest for obs-asio on the winget-pkgs repository (winget is the windows package manager) and in order to do upgrades, obs-asio would have to have a version number. 
Issue in winget-pkgs:
https://github.com/microsoft/winget-pkgs/issues/16070#issuecomment-910433668

**Additional context**
The winget-cli project: https://github.com/microsoft/winget-cli

**Comment by Andersama** [comment]
@leochras I've never heard of winget-pkgs and I'm definitely not going to be maintaining it, you're better off asking whoever added the manifest how to get it working. Also, the plugin rarely updates.

**Comment by vedantmgoyal2009** [comment]
What you need to do is to add a version in the installer you make. Nothing else.

**Comment by Andersama** [comment]
Ah, well ok, that's not necessarily going to help anyone then because the version numbers for this "plugin" don't make much sense anyway. There are after all two distinct versions really, and only one of which I decided to keep going. First I guess I have to ask which version is winget-pkgs currently trying to install? I'm going to assume it's the most recent Juce based one.

**Comment by Andersama** [comment]
How exactly would this work without updating the manifest for you guys? It seems as though this is just executing the installer specified at the url. Even if I updated the installer, the next installer wouldn't be at that url, how would winget-pkgs know there was any update?

**Comment by vedantmgoyal2009** [comment]
Even if I updated the installer, the next installer wouldn't be at that url, how would winget-pkgs know there was any update?

We will update the Installer Url.

**Comment by Andersama** [comment]
So this isn't automated at all? It's entirely user maintained?

**Comment by Andersama** [comment]
Why is the version number necessary if the url has to be updated?

**Comment by leochras** [comment]
Yes, it is user maintained, and we will have to update the installer link if there **was** a new version. Winget is still in development and therefore not perfect. The developer only has to consider (maybe for the next update) telling the installer to set a registry key with the version number, so Windows knows which version of the program is installed on the system, therefore Winget can check if there is a new version available.

**Comment by parkerauk** [comment]
Hello, the main reason for asking is to avoid millions of your customers having delays when running WINGET to update their PCs.  The upgrade routine runs every time, when it should not. Core OBS is fine, it is just the ASIO that is the issue.
